### DISCUSSION TITLE
Lesson 7: Error:  (code: -32000, message: Must be authenticated!, data: None)

### DISCUSSION THREAD

**Discussion by jobmwitah** [Cyfrin/foundry-full-course-cu#105] [Q&A] [answered]
I am Trying to deploy my code using make and I am getting this error

```makefile
Error: 
(code: -32000, message: Must be authenticated!, data: None)
make: *** [Makefile:43: deploy] Error 1
```
Could anyone help me find a way out of this Please??😥. The code deploys correctly on anvil.

**Comment by PatrickAlphaC** [comment, marked as answer]
Yep! We've seen this [issue before](https://stackoverflow.com/questions/70689587/hardhat-and-rinkeby-providererror-must-be-authenticated-error)

Usually it means there is an issue with your RPC URL.

**Reply by jobmwitah** [reply]
Thanks Boss It Worked
